lib/services/azurerediscache/index.js

```diff
@@ -110,7 +110,7 @@ Handlers.bind = function (params, next) {
       
       var redisUrl = util.format('redis://%s:%s@%s:%s',
                                   provisioningResult.name,
-                                  accessKeys.primaryKey,
+                                  accessKeys.primaryKey.replace(/\//g, '%2F'),
```

Comment:
Shall we use a library to urlencode the primary key instead of simply replacing?